python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -260,18 +298,18 @@ def resolution_matching(jet, genjet):
             jet_pt_orig = jet_pt
             rawFactor = jet.rawFactor
 
-            #redo JECs if desired
             if hasattr(jet, "rawFactor"):
                 jet_rawpt = jet_pt * (1 - jet.rawFactor)
                 jet_rawmass = jet_mass * (1 - jet.rawFactor)
             else:
                 jet_rawpt = -1.0 * jet_pt #If factor not present factor will be saved as -1
                 jet_rawmass = -1.0 * jet_mass #If factor not present factor will be saved as -1
 
-            (jet_pt, jet_mass)    = self.jetReCalibrator.correct(jet,rho)
-            (jet_pt_l1, jet_mass_l1) = self.jetReCalibratorL1.correct(jet,rho)
+            (jet_pt, jet_mass) = self.jetReCalibrator.correct(jet,rho)
             jet.pt = jet_pt
             jet.mass = jet_mass
+            
```

Comment:
What is the reasoning behind the change in order here? Previously, `jetRecalibrator1` was called *before* `jet.pt` was set, now it's called *after* `jet.pt` is set. Doesn't that make a difference (at least when you're switching JES)?

Reply:
Unintented (had made another change and only partly reverted it)... But now I understand the issue with the MET: while this had no effect on regular jets if the same JES is used, it did mess up the low-pt jets and hence the corrected MET.

Now I get perfect agreement with the previous version of the code also for the MET.